src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperOutput.cs

```diff
@@ -21,17 +24,32 @@ internal TagHelperOutput(string tagName)
         /// </summary>
         /// <param name="tagName">The HTML element's tag name.</param>
         /// <param name="attributes">The HTML attributes.</param>
+        public TagHelperOutput(string tagName, TagHelperAttributeList attributes)
```

Comment:
Can we remove this constructor? If not at least make it `internal` (for testing).